src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/App/index.unit.spec.js

```diff
@@ -2,33 +2,14 @@
 import React from 'react';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
+import CallToActionWidget from 'applications/static-pages/cta-widget';
 // Relative imports.
-import AuthContent from '../AuthContent';
-import UnauthContent from '../UnauthContent';
 import { App } from './index';
 
-describe('Get Medical Records Page <App>', () => {
-  it('renders what we expect when not a Cerner patient', () => {
-    const wrapper = shallow(
-      <App
-        showNewScheduleViewAppointmentsPage
-        facilities={[{ usesCernerAppointments: false }]}
-      />,
-    );
-    expect(wrapper.find(UnauthContent)).to.have.lengthOf(1);
-    expect(wrapper.find(AuthContent)).to.have.lengthOf(0);
-    wrapper.unmount();
-  });
-
-  it('renders what we expect when a Cerner patient', () => {
-    const wrapper = shallow(
-      <App
-        showNewScheduleViewAppointmentsPage
-        facilities={[{ usesCernerAppointments: true }]}
-      />,
-    );
-    expect(wrapper.find(UnauthContent)).to.have.lengthOf(0);
-    expect(wrapper.find(AuthContent)).to.have.lengthOf(1);
+describe('Appointments Page <App>', () => {
```

Comment:
I expect that CallToActionWidget should have its own tests, so not much to test here